spec/models/entity_group_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe EntityGroup, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] Kindly remove the tests if you do not need them. To make your code cleaner.